ha/remote_execution/ssh_communication.py

```diff
@@ -0,0 +1,52 @@
+import paramiko
+
+from cortx.utils.ssh import SSHChannel
+from ha.remote_execution.remote_executor import RemoteExecutor
+from ha.const import RA_LOG_DIR
+from cortx.utils.log import Log
+
+class SSHRemoteExecutor(RemoteExecutor):
+    '''class which enables remote communication using SSH'''
+
+    def __init__(self, hostname, username, port=22, password=None):
```

Comment:
username can also be None.
The default username would be the current user.

Reply:
Agree

Reply:
I have kept it as None